cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -367,7 +393,8 @@ private static String wrapQueryExpression(String term, Index index) {
   }
 
   @SuppressWarnings("squid:S135")  // suppress "reduce to one continue in for loop"
-  private SqlSelect toSql(CQLSortNode node) throws QueryValidationException {
+  private SqlSelect toSqlSort(CQLSortNode node, int offset, int limit, String table, String column)
```

Comment:
Can you remove `private` and write some unit tests to cover the border cases of limit and offset?